samples/subsys/llext/fs_loader/makeimg.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# Create a FAT12 disk image to flash in memory
+# This tool requires the following to be available on the host system:
+#
+# - dosfstools
+# - mtools
+
+OUTPUT=$1
+INPUTS=${@:2}
+LOGICAL_SECTOR_SIZE=512
+DISK_SIZE_KB=128
+
+if [ $# -lt 2 ]; then
+    echo -e "Not enough arguments.\nUsage:\n\t`basename $0` output.img input1 [input2] [...]"
```

Comment:
Avoid `echo` (and use shellcheck)
```suggestion
    printf 'Not enough arguments.\nUsage:\n\t%s output.img input1 [input2] [...]' "$0"
```

Or much more readable for long strings:
```
cat <<EOFUSAGE
Not enough arguments.
Usage:
    $0 output.img input1 [input2] [...]
EOFUSAGE